gui/GiveItem.java

```diff
@@ -82,19 +92,9 @@ private void giveItemToPlayer() {
 		if (itmSt != null) {
 			itemMod = " " + itmSt.getItemDamage();
 		}
-		String amt;
-		if (!((amountBox.getText() == null) || amountBox.getText().isEmpty())) {
-			try {
-				int amount = Math.abs(Integer.parseInt(amountBox.getText())) % 65;
-				amt = "" + amount;
-			} catch (NumberFormatException nfe) {
-				amt = "1";
-			}
-		} else {
-			amt = "1";
-		}
-		Minecraft.getMinecraft().thePlayer.sendChatMessage(
-				"/give " + userBox.getText() + " " + tItem.getRegistryName() + " " + amt + " " + itemMod);
+		String amt = amountBox.getText() == null ? "1" : amountBox.getText();
```

Comment:
also check for an empty string so if you want to do a ternary operator make it like 

(amountBox.getText() == null) || amountBox.getText().isEmpty()) ? "1" : amountBox.getText();